load-tester/src/launch_node_and_rpc.rs

```diff
@@ -0,0 +1,182 @@
+use std::io;
+use std::fs;
+use std::fs::File;
+use std::path::Path;
+use std::process::Command;
+
+use serde_json;
+
+use futures::Future;
+use hyper::client::Connect;
+
+use tokio_core::reactor::Handle;
+use tokio_process::{Child, CommandExt};
+
+use serde_json::Value;
+
+use errors::*;
+
+use rpc::RpcClient;
+
+const RPC_PORT_START: u64 = 55000;
+const PEERING_PORT_START: u64 = 54000;
+const IPC_PORT_START: u64 = 56000;
+
+pub fn launch_node_and_rpc(
+    nano_node: &Path,
+    nano_rpc: &Path,
+    tmp_dir: &Path,
+    handle: Handle,
+    i: u64,
+) -> Result<(Child, Child, RpcClient)> {
+    let data_dir = tmp_dir.join(format!("Nano_load_test_{}", i));
+    match fs::create_dir(&data_dir) {
+        Ok(_) => {}
+        Err(ref e) if e.kind() == io::ErrorKind::AlreadyExists => {
+            let _ = fs::remove_file(data_dir.join("data.ldb"));
+            let _ = fs::remove_file(data_dir.join("wallets.ldb"));
+        }
+        r => r.chain_err(|| "failed to create nano_node data directory")?,
+    }
+    let peering_port = PEERING_PORT_START + i;
+    let ipc_port = IPC_PORT_START + i;
+
+    let config = json!({
+    "version": "2",
+    "rpc_enable": "false",
+    "rpc": {
+        "version": "1",
+        "enable_sign_hash": "false",
+        "max_work_generate_difficulty": "ffffffffc0000000",
+        "rpc_path": "C:\\Users\\Wesley\\Documents\\nano-node\\build\\Debug\\nano_rpc.exe",
```

Comment:
Probably want a placeholder here

Reply:
Yeh that looks weird :D, it's not actually used here as we are using out of process RPC (passed in via command line), so I've just removed it.